src/main/java/org/embulk/output/mailchimp/MailChimpAbstractRecordBuffer.java

```diff
@@ -267,15 +267,17 @@ public JsonNode apply(JsonNode input)
                 property.put("status", task.getDoubleOptIn() ? PENDING.getType() : SUBSCRIBED.getType());
                 ObjectNode mergeFields = JsonNodeFactory.instance.objectNode();
                 for (String allowColumn : allowColumns.keySet()) {
-                    String value = input.findValue(allowColumns.get(allowColumn)).asText();
+                    JsonNode field = input.findValue(allowColumns.get(allowColumn));
+                    String value = field != null ? field.asText() : "";
```

Comment:
You can also use
* `input.hasNonNull(allowColumns.get(allowColumn))` 
* `field.isNull()`